code_samples/client_statistics.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var Client = require('hazelcast-client').Client;
+var ClientConfig = require('hazelcast-client').Config.ClientConfig;
+
+function createConfig() {
+    var cfg = new ClientConfig();
+    cfg.properties['hazelcast.client.statistics.enabled'] = true;
+    cfg.properties['hazelcast.client.statistics.period.seconds'] = 2;
+    return cfg;
+}
+
+Client.newHazelcastClient(createConfig()).then(function (client) {
+    client.shutdown();
```

Comment:
you may want to let the client sleep or wait here until the statistics is populated and tell the user to go an check at management center for the stats. (Check example https://github.com/hazelcast/hazelcast-cpp-client/blob/master/examples/client-statistics/ClientStatistics.cpp)

Reply:
Thanks for the good sample. I updated node.js sample to do similar actions. Can you take another look @ihsandemir ?